src/_includes/cookie-notice.html

```diff
@@ -0,0 +1,10 @@
+<section class="cookie-notice">
+  <div class="container">
+    <p>Google uses cookies to deliver its services, to personalize ads, and to 
+      analyze traffic. You can adjust your privacy controls anytime in your 
+      <a href="https://myaccount.google.com/data-and-personalization" target="_blank" rel="noopener">Google settings</a>. 
+      <a href="https://policies.google.com/technologies/cookies" target="_blank" rel="noopener">Learn more</a>.
```

Comment:
@sfshaza2  How about something like "To learn more, see [How Google uses cookies](https://policies.google.com/technologies/cookies)."

Reply:
It was requested by @RedBrogdon that we carry over the same cookie notice module from Flutter.dev, which has the approved messaging and linkage from legal. Happy to change messaging if that is the consensus. 

I can add aria, etc to the button. I changed the styles of the banner on the homepage only so that it matches those particular styles, so it makes sense to be complete here. I'll add that.

Reply:
If that is the message from Andrew and legal, then that is perfectly fine! Removing the external link icons will make it much more manageable.

Reply:
@parlough I have pushed up those style/accessibility changes.